### PR TITLE
Add Subsistence

### PR DESCRIPTION
**Pull request by Th3Dilli**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

### New Server Submissions:

1. [x] Does your submission pass tests (server is connectable)?
2. [ ] Does your server use a custom docker image?
3. [x] Have you added the server to the main README.md?
4. [x] Have you added a unique README.md for the server you are adding?

### CODE REVIEW

File: game_eggs/steamcmd_servers/subsistence/egg-subsistence-wine.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            "container": "debian:buster-slim",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "STEAM_USER",
```
Comment (Software-Noob) [inline game_eggs/steamcmd_servers/subsistence/egg-subsistence-wine.json:32]: A user without a password is not useful. If the game doesn't require steam auth, you don't have to provide a variable.

File: game_eggs/steamcmd_servers/subsistence/egg-subsistence-wine.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            "description": "",
+            "env_variable": "STEAM_USER",
+            "default_value": "anonymous",
+            "user_viewable": false,
+            "user_editable": true,
+            "rules": "required|string"
+        },
+        {
+            "name": "INSTALL_FLAGS",
```
Comment (Software-Noob) [inline game_eggs/steamcmd_servers/subsistence/egg-subsistence-wine.json:41]: You should provide the SRCD_APPID and AUTO_UPDATE variables for the installation and auto-update. The INSTALL_FLAGS only applies to the install script and is currently ignored in the docker entrypoint - something we should take care of.

These are covered in our example [steamcmd install script](https://github.com/parkervcp/eggs/blob/master/scripts/steamcmd_installer.sh)

File: game_eggs/steamcmd_servers/subsistence/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Subsistence Wine
+
+Subsistence is a sandbox, first person, solo or co-op, PvE open-world survival game. Struggle through changing seasons to build a base, develop technology and gear-up in the hostile environment. Defend yourself from wildlife, the elements and AI hunters (who also build bases in the world).
+
+`UDKGame/Config/UDKDedServerSettings.ini`
```
Comment (Software-Noob) [inline game_eggs/steamcmd_servers/subsistence/README.md:5]: This by itself doesn't tell us anything

**Comment by Th3Dilli** [comment]
Thank you for your feedback.

I should have adressed everything now

File: game_eggs/steamcmd_servers/subsistence/egg-subsistence-wine.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-12-20T02:19:34-05:00",
+    "name": "Subsistence Wine",
+    "author": "th3dilli@gmx.at",
+    "description": "a Subsistence Wine egg",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_wine-5"
```
Comment (Software-Noob) [inline game_eggs/steamcmd_servers/subsistence/egg-subsistence-wine.json:13]: Please migrate to our new docker images https://github.com/parkervcp/yolks#wine

Comment (Th3Dilli) [reply game_eggs/steamcmd_servers/subsistence/egg-subsistence-wine.json:13]: i did try today but ran in to some issues

will take another look at it soon

**Comment by Th3Dilli** [comment]
hey sorry for no update in some time

it seems very unstable with wine and I am not sure if it is worth the effort
also the community that wanted this egg does not need it anymore and so I wouldn't continue on this

if some one else wants to take over this egg feel free to do so
and if there is no demand or the next person should open a new PR feel free to close this

thanks for the reviews and your time @Software-Noob

**Comment by QuintenQVD0** [comment]
it should run fine with proton.

**Comment by QuintenQVD0** [comment]
@Th3Dilli I got it to sort of start but it has a empty log file and no console. can you test it if I send it to you?

**Comment by Th3Dilli** [comment]
yeah can do that @QuintenQVD0

**Comment by QuintenQVD0** [comment]
For those reading this, it is still on my to-do list to make this run with proton.

**Pull request closed without merging**